donate.html

```diff
@@ -18,6 +18,9 @@ <h1><span class="main-headline">Donations</span></h1>
         <input type="number" id="donation-amount" class="form-control" value="25" aria-label="Amount (to the nearest dollar)">
         <label for="special-instructions">Special instructions</label>
         <input type="text" class="form-control" id="special-instructions" placeholder="Special instructions">
+        <input type="checkbox" id="cover-fees" />
+        <label for="cover-fees">Cover processing fees (2.2% + $0.30)</label>
```

Comment:
is it just 2.2%? or 3%? i know that is what we include on our pledge form 🤔

Reply:
The typical rate is 2.9%, but our non-profit rate is 2.2% + $0.30.

However, in checking this out, I realized that Stripe has been charging your account at the typical rate instead of the non-profit rate. I've asked them to fix that and to refund the difference between the rates.

Reply:
omg amazing, thank youu. we'll also need to update our pledge form to match that.